src/api/database/bulk-api.rst

```diff
@@ -147,6 +147,139 @@
             "offset" : 0
         }
 
+Sending multiple queries to a database
+======================================
+
+.. versionadded:: 2.1+
+
+.. http:post:: /{db}/_all_docs
+    :synopsis: Returns results for the specified queries
+
+    Executes multiple specified built-in view queries of all documents in this
+    database. This enables you to request multiple documents in a single
```

Comment:
I wonder if it would be worth explaining in more detail that (as I understand) this allows combining the results of _multiple queries_, not just _multiple documents_ (`_all_docs` already returns multiple documents, so I can see room for confusion, upon a cursory read of this statement).  Perhaps something like:

> This enables you to request the results of multiple queries in a single request, in place of ...